services/terrain/src/terrain/services/permanent_id_requests.clj

```diff
@@ -355,4 +379,5 @@
   (let [identifier (complete-permanent-id-request (:shortUsername current-user)
                                                   (admin-get-permanent-id-request request-id nil))]
     (update-permanent-id-request request-id nil (json/encode {:status       status-code-completion
+                                                              :comments     identifier
```

Comment:
Oh, I see, this is what's going into the body of the email. Why not just use the pre-existing `:permanent_id` key, then?